config/ember-try.js

```diff
@@ -68,6 +71,7 @@ module.exports = function() {
           name: 'ember-lts-2.12',
           npm: {
             devDependencies: {
+              'ember-cli-htmlbars-inline-precompile': '~1.0.5',
```

Comment:
I don't think we need any modification to this one (since 2.12 already comes from npm and is supported by 2.0.0 of ember-cli-htmlbars-inline-precompile).